packages/atauth/include/atauth/cjson.h

```diff
@@ -0,0 +1,10 @@
+#ifndef ATCLIENT_CJSON_H
+#define ATCLIENT_CJSON_H
```

Comment:
define should be atauth_cjson, but also atclient is a dependency of atauth, no need to duplicate this file, just `#include atclient/cjson.h`